src/plugins/intel_cpu/src/cpu_streams_calculation.cpp

```diff
@@ -160,5 +164,131 @@ std::vector<std::vector<int>> get_streams_info_table(const int input_streams,
         }
     }
 }
+
+int get_model_prefer_threads(const int num_streams,
```

Comment:
For latency mode, just need model_prefer_threads for hybrid platform.

Reply:
Ok, done.